packages/message-hash/src/index.ts

```diff
@@ -14,14 +19,30 @@ export function messageHash(
   const contentTopicBytes = utf8ToBytes(message.contentTopic);
 
   let bytes;
-
-  if (message.meta) {
+  if (message.meta && message.timestamp) {
```

Comment:
we can simplify it like this:
```ts
const bytes = concat([
pubsubTopicBytes,
      message.payload,
      contentTopicBytes,
      message.meta,
      numberToBytes(message.timestamp)
].filter(isDefined));
```

Reply:
[0d786eb](https://github.com/waku-org/js-waku/pull/1986/commits/0d786ebf574389d78f628d522e216b1466789588)